static/css/v2-announcement.css

```
@@ -54,19 +54,21 @@
+    background: linear-gradient(90deg, rgba(255, 255, 255, 0.06), rgba(124, 58, 237, 0.9), rgba(99, 102, 241, 0.9), rgba(255, 255, 255, 0.06));
     background-size: 200% 100%;
     background-position: 0% 50%;
     pointer-events: none;
     z-index: 1;
     /* show only the band by using mask with content-box */
-    -webkit-mask: linear-gradient(#fff,#fff) content-box, linear-gradient(#fff,#fff);
+    -webkit-mask: linear-gradient(#fff, #fff) content-box, linear-gradient(#fff, #fff);
     -webkit-mask-composite: xor;
     mask-composite: exclude;
-    mask: linear-gradient(#fff,#fff) content-box, linear-gradient(#fff,#fff);
+    mask: linear-gradient(#fff, #fff) content-box, linear-gradient(#fff, #fff);
```

Stylelint (17.6.0)

[error] Unexpected shorthand "mask" after "mask-composite" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)